pandaserver/taskbuffer/OraDBProxy.py

```diff
@@ -228,6 +228,8 @@ def querySQLS(self,sql,varMap,arraySize=1000):
             self.conn.begin()
             self.cur.arraysize = arraySize
             ret = self.cur.execute(sql+comment,varMap)
+            if ret:
+                ret = True
```

Comment:
I'd like to bring this change to your notice.

The "ret" here used to be a cx_Oracle.Cursor object if query succeeded, or to be False if query failed.

It was not possible to run this querySQLS method with taskBufferInterface since cx_Oracle.Cursor object is not picklable to share between processes.

My change should be reasonable, while I did not check every other line that querySQLS (but in many functions, "ret" is never used after querySQLS called)

Let me know if there is any concern.